main.nf

```diff
@@ -96,6 +105,11 @@ if (params.verbose){
 
 reference_fasta_index = "${params.reference_fasta}.fai"
 
+// Collect GRIDSS reference files
+gridss_reference = Channel.fromPath( "${params.gridss_reference_dir}/*", checkIfExists: true )
+gridss_reference_fasta = gridss_reference.filter { it.name.endsWith(".fasta") }.collect()
```

Comment:
**comment (blocking)**: We shouldn't really be hard-coding this, FASTA files can have other extensions like `.fa` that wouldn't work with this. I suggest accepting the FASTA file path and then inferring the reference directory from the given path and then making sure to note that the path to the FASTA that needs to be given is the one in the GRIDSS reference directory.